src/styles/images/index.md.njk

```diff
@@ -8,53 +8,70 @@ layout: layout-pane.njk
 
 {% from "_example.njk" import example %}
 
-Avoid unnecessary decoration. Only use images if there’s a real user need.
+Avoid using images for unnecessary decoration. Only use images if there’s a real user need.
 
-## Aspect ratio
+This guidance is for government teams that build online services. To learn how to use images in GOV.UK mainstream content, go to [GOV.UK content guidance for images](https://www.gov.uk/guidance/content-design/images).
 
-### Photography
+## Using images to give information
 
-The aspect ratio for photographs should be 3:2.
+Services usually work best without relying on images. Focus on writing clear, simple content for your service first.
 
-{{ example({group: "styles", item: "images", example: "default", html: false, open: false, size: "l"}) }}
+Adding images can sometimes help users better understand instructions. However, images should only be visual aids — written content must contain all the information the user needs to complete the service.
 
+If you choose to add an image to your service, you’ll need to carefully decide if the image needs to be described with alternative text (alt text).
 
-### Illustrations or representative imagery
+### Alt text
 
-If your image represents something physical, such as a letter, document or credit card you should use the aspect ratio of that item.
+Alternative text, or alt text, is read out by screen readers or displayed if an image does not load or if images have been switched off.
```

Comment:
```suggestion
Alt text is read out by screen readers, or displayed, if:

- an image does not load
- images have been switched off

```

Reply:
Hmmm yeah, I see what you mean...

The first time I read the sentence, I thought "if an image does not load or if images have been switched off" only applied to "displayed" and not to "read out by screen readers". (This is probably me being dozy, though.)

I don't think there's anything wrong with the original wording, but here might be another way of putting it:

> If an image does not load or if images have been switched off, alt text is read out by screen readers or displayed onscreen.

Reply:
Another idea? 
I agree that maybe the concepts in this sentence need to be explained a bit clearer.
This sentence is a definition of 'alt text' so would like to try to keep it as the subject.

```suggestion
Alternative text, or alt text, is read out by screen readers or displayed onscreen — usually when images do not load or have been disabled in browser settings.
```

Reply:
Great suggestion!